### PR TITLE
Feat: add loess model

### PR DESCRIPTION
**Pull request by jennifer-hoang**
- Adds loess model to analysis script
- Adds additional EDA with Tweedie, modified Gamma
- Updates Makefile, dependencies

### CODE REVIEW

**Review by shyan0903 — COMMENTED** [review]
Looks good to me!

File: src/eda/04_modeling_eda_tweedie.Rmd [hunk 1]
```diff
@@ -36,8 +36,8 @@ When p = 1, the distribution is equivalent to a Poisson distribution.
 When p = 2, the distribution is equivalent to a gamma distribution.
```
Comment (mahm00d27) [inline src/eda/04_modeling_eda_tweedie.Rmd:36]: Can be mentioned here (For the readers), that "p" here, is not "p-value", but rather an argument to determine the model, in R-INLA package.

Comment (jennifer-hoang) [reply src/eda/04_modeling_eda_tweedie.Rmd:36]: This was defined in line 31, "index parameter (p)". Thanks!

**Review by mahm00d27 — APPROVED** [review]
Reviewed, commented and approved

**Pull request merged**